quickstart/examples/wide-ep-lws/README.md

```diff
@@ -1,11 +1,140 @@
 # Well-lit Path: Wide Expert Parallelism (EP/DP) with LeaderWorkerSet
 
+## Warning ⚠️: Here there be Dragons
```

Comment:
this is not true right? I thought we just wanted to do one-pod-per-node?

cc @tlrmchlsmth